packages/lit/src/index.max.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import './polyfill-support.js';
+
+export * from './index.js';
+
+export * from './async-directive.js';
+export * from './decorators.js';
+export * from './directive-helpers.js';
+export * from './directive.js';
+export * from './directives/async-append.js';
+export * from './directives/async-replace.js';
+export * from './directives/cache.js';
+export * from './directives/choose.js';
+export * from './directives/class-map.js';
+export * from './directives/guard.js';
+export * from './directives/if-defined.js';
+export * from './directives/join.js';
+export * from './directives/keyed.js';
+export * from './directives/live.js';
+export * from './directives/map.js';
+export * from './directives/range.js';
+export * from './directives/ref.js';
+export * from './directives/repeat.js';
+export * from './directives/style-map.js';
+export * from './directives/template-content.js';
+export * from './directives/unsafe-html.js';
+export * from './directives/unsafe-svg.js';
+export * from './directives/until.js';
+export * from './directives/when.js';
+export {
+  html as staticHtml,
+  literal,
+  svg as staticSVG,
+  unsafeStatic,
+  withStatic,
+} from './static-html.js';
```

Comment:
Rollup was nice enough to complain that re-exporting everything from this file as-is would cause a collision. I don't think there's a way to say "re-export everything, but rename just these specific things", so I had to list them all out here. Fortunately there's not many.

Reply:
Leave a comment here and in lit-html/static.js that this export needs to be updated if new exports are added to that file.